core/src/main/java/com/scalar/db/common/CoreError.java

```diff
@@ -678,6 +678,24 @@ public enum CoreError implements ScalarDbError {
       "Mutations across multiple storages are not allowed. Mutations: %s",
       "",
       ""),
+  COLUMN_SPECIFIED_AS_PRIMARY_KEY(
+      Category.USER_ERROR,
+      "0216",
+      "The column is specified as a primary key. Table: %s; Column: %s",
+      "",
+      ""),
+  COSMOS_DROP_COLUMN_NOT_SUPPORTED(
+      Category.USER_ERROR,
+      "0217",
+      "Drop column functionality is not supported in Cosmos DB",
+      "",
+      ""),
+  DYNAMO_DROP_COLUMN_NOT_SUPPORTED(
+      Category.USER_ERROR,
+      "0218",
+      "Drop column functionality is not supported in DynamoDB",
```

Comment:
Ditto.
```suggestion
      "DynamoDB does not support the dropping column feature",
```
Or, if we expect more databases or data stores that don't support the dropping column feature, it might be better to just have one message like "The storage does not support the dropping column feature".